Documentation/ColumnsConfig/Type/inputDateTime.rst

```diff
@@ -143,6 +143,12 @@ readOnly
 search
 ------
 
+.. note::
+
+    Input fields of renderType ``date``, ``time`` and ``int`` are excluded from search by default.
```

Comment:
There are no `renderType` options like these by default. Did you mean `eval`? Also this should be mentioned in general for `input` since there is no `renderType` check actually.

Reply:
I removed it from this position in favor of the hint in `CtrlSearchFields.rst`